packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -331,7 +345,12 @@ export class Dropdown {
 
   private close(event?: Event) {
     if (event?.defaultPrevented) {
-      return;
+      const target = event.target as HTMLElement;
+      if (
+        target.shadowRoot.contains(this.anchorElement) ||
```

Comment:
Does this always work? What happend target element is element without an shadowDOM?